workflow-templates/pr-announcer.yml

```diff
@@ -0,0 +1,29 @@
+name: "Google Chats PR Announcer"
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Every morning at 6AM, Mondays to Fridays
+    - cron: "0 6 * * MON-FRI"
+
+jobs:
+  prnouncer:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: guardian/actions-prnouncer@main
+        with:
+          # https://user-images.githubusercontent.com/21217225/191922153-ae6fcebe-ca87-4611-93ee-0e66a47dab5d.png
+          # Anybody can create a new Webhook for a google chatroom, press the caret by the Space name and press "Manage Webhooks"
+          # You can use any value for the name and avatar URL. You should add this secret to your repository settings.
+          google-webhook-url: ${{ secrets.GOOGLE_WEBHOOK_URL }}
+
+          # Defaults to the repository that this action is configured in.
+          # You can specify multiple repositories by using a comma delimited string!
+          # github-repositories: dotcom-rendering,frontend
```

Comment:
This is neat! What's the best place for this workflow to live when monitoring multiple repos? A `*-platform` based repo, maybe?

Reply:
Do you mean a repo for the entire department? Or one per team? I _think_ I'd favour the former.

Reply:
I think we could get the department to place configs in https://github.com/guardian/prnouncer-config. WDYT?